src/openrct2/ride/rtd/coaster/LSMLaunchedRollerCoaster.h

```diff
@@ -0,0 +1,90 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2024 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "../../../sprites.h"
+#include "../../RideData.h"
+#include "../../ShopItem.h"
+#include "../../Track.h"
+
+// clang-format off
+constexpr RideTypeDescriptor LSMLaunchedRollerCoasterRTD =
+{
+    .Category = RIDE_CATEGORY_ROLLERCOASTER,
+    .StartTrackPiece = OpenRCT2::TrackElemType::EndStation,
+    .TrackPaintFunctions = TrackDrawerDescriptor({
+        .Drawer = GetTrackPaintFunctionLatticeTriangleAlt,
+        .supportType = MetalSupportType::Tubes,
+        .EnabledTrackPieces = {TrackGroup::straight, TrackGroup::stationEnd, TrackGroup::flatRollBanking, TrackGroup::slope, TrackGroup::slopeSteepUp, TrackGroup::slopeSteepDown, TrackGroup::slopeCurve, TrackGroup::slopeCurveSteep, TrackGroup::sBend, TrackGroup::curveSmall, TrackGroup::curve, TrackGroup::curveLarge, TrackGroup::helixDownBankedHalf, TrackGroup::helixUpBankedHalf, TrackGroup::brakes, TrackGroup::onridePhoto, TrackGroup::blockBrakes, TrackGroup::slopeRollBanking, TrackGroup::slopeSteepLong, TrackGroup::slopeCurveBanked, TrackGroup::slopeCurveLarge, TrackGroup::slopeCurveLargeBanked, TrackGroup::booster, TrackGroup::poweredLift, TrackGroup::slopeVertical, TrackGroup::curveVertical, TrackGroup::barrelRoll, TrackGroup::quarterLoop, TrackGroup::halfLoop, TrackGroup::halfLoopMedium, TrackGroup::halfLoopLarge, TrackGroup::verticalLoop, TrackGroup::corkscrew, TrackGroup::corkscrewLarge, TrackGroup::zeroGRoll, TrackGroup::zeroGRollLarge },
+        .ExtraTrackPieces = {TrackGroup::liftHill, TrackGroup::liftHillCable},
+        }),
+    .InvertedTrackPaintFunctions = {},
+    .Flags = kRtdFlagsHasThreeColours | kRtdFlagsCommonCoaster | kRtdFlagsCommonCoasterNonAlt |
+                 EnumsToFlags(RtdFlag::hasLeaveWhenAnotherVehicleArrivesAtStation, RtdFlag::checkGForces, 
+                              RtdFlag::allowMultipleCircuits, RtdFlag::allowCableLiftHill, RtdFlag::allowReversedTrains),
+    .RideModes = EnumsToFlags(RideMode::ContinuousCircuit, RideMode::ContinuousCircuitBlockSectioned, RideMode::PoweredLaunch, RideMode::PoweredLaunchBlockSectioned),
+    .DefaultMode = RideMode::ContinuousCircuit,
+    .OperatingSettings = { 2, 10 },
+    .TrackSpeedSettings = { 60, 60 },
+    .BoosterSettings = { 17, 68 },
+    .LegacyBoosterSettings = { 17, 68, 4 },
```

Comment:
Yes, these will work.

Reply:
We should probably copy the booster values from the Giga Coaster, though. Currently, those can be set up to 215 km/h, while these only go to 107 km/h. What do you reckon?

Reply:
I used to have those copied, but I changed them after some feedback. We're better off having smaller steps since it hits the highest point with this speed anyways.